rs/ethereum/cketh/minter/src/eth_logs/mod.rs

```diff
@@ -424,3 +511,27 @@ fn parse_principal_from_slice(slice: &[u8]) -> Result<Principal, String> {
     }
     Principal::try_from_slice(principal_bytes).map_err(|err| err.to_string())
 }
+
+enum InternalLedgerSubaccountTag {}
+type InternalLedgerSubaccount = CheckedAmountOf<InternalLedgerSubaccountTag>;
+
+/// Ledger subaccount.
+///
+/// Internally represented as a u256 to optimize cbor encoding for low values,
```

Comment:
Again just for my understanding, why is a `u256` representation an optimization?

Reply:
It's because of how we do the cbor [serialization](https://github.com/dfinity/ic/blob/9fb40537484e9fd4d575e0b6d16cb164afab8b07/rs/ethereum/cketh/minter/src/cbor/u256.rs#L37).